docs/en/latest/config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "version": 1.0,
+    "sidebar": [
+      {
+        "type": "category",
+        "label": "Installation",
+        "items": [ "development", "how-it-works", "the-internal-of-apisix-java-plugin-runner" ]
```

Comment:
IMHO, each item should have separate sidebar entry, and they are not all about "Installation".

Reply:
Okay captain